mixins/kubernetes/config.libsonnet

```diff
@@ -37,19 +37,19 @@
       'apiserver.json': std.md5('apiserver.json'),
       'cluster-total.json': std.md5('cluster-total.json'),
       'controller-manager.json': std.md5('controller-manager.json'),
-      'k8s-resources-cluster.json': 'efa86fd1d0c121a26444b636a3f56738',
```

Comment:
Hi Vishwa, should we be making sure these ids match? 
https://msazure.visualstudio.com/AzureGrafanaService/_git/ResourceProvider/pullrequest/10855454?_a=files&path=/src/Grafana.Content/AzureMonitoringAccountIntegrationDashboards/k8s-resources-cluster.json
Or is there a list of ids in grafana repo that we can compare this with?

Reply:
The IDs should match from this location (https://msazure.visualstudio.com/AzureGrafanaService/_git/ResourceProvider?path=/src/Grafana.DP.GrafanaContainer/provision/oob-dashboards/mac-aks) . This way, next time, when we build our dashboards they all will have the IDs we have shipped automatically and we don't have to manually edit them.